test/unit/metadata.js

```diff
@@ -58,6 +58,19 @@ describe('Image metadata', function () {
     });
   });
 
+  it('JPEG with IPTC/XMP', function (done) {
+    sharp(fixtures.inputJpgWithIptcAndXmp).metadata(function (err, metadata) {
+      if (err) throw err;
+      // IPTC
+      assert.strictEqual('object', typeof metadata.iptc);
+      assert.strictEqual(true, metadata.iptc instanceof Buffer);
+      // XMP
```

Comment:
Given there's no well-maintained IPTC parser, perhaps we could add an assertion on the Buffer length?

Reply:
This can work for now. Thanks for the suggestion @lovell! I will add it.

Reply:
Looks like we also can test it by checking that buffer starts with `Photoshop` word.

```
⌘  exiv2 -pS test/fixtures/Landscape_9.jpg
STRUCTURE OF JPEG FILE: test/fixtures/Landscape_9.jpg
 address | marker       |  length | data
       0 | 0xffd8 SOI
       2 | 0xffe0 APP0  |      16 | JFIF............
      20 | 0xffe1 APP1  |   18835 | Exif..II*...............z......
   18857 | 0xffe1 APP1  |   12497 | http://ns.adobe.com/xap/1.0/.<?x
   31356 | 0xffed APP13 |   18252 | Photoshop 3.0.8BIM..............
   49610 | 0xffe2 APP2  |    2444 | ICC_PROFILE......|lcms.0..mntrRG chunk 1/1
   52056 | 0xffdb DQT   |      67
   52125 | 0xffdb DQT   |      67
   52194 | 0xffc0 SOF0  |      17
   52213 | 0xffc4 DHT   |      28
   52243 | 0xffc4 DHT   |      67
   52312 | 0xffc4 DHT   |      27
   52341 | 0xffc4 DHT   |      49
   52392 | 0xffda SOS
```